ansible/library/topo_facts.py

```diff
@@ -285,6 +285,12 @@ def get_topo_config(self, topo_name, hwsku, testbed_name, asics_present, card_ty
             vm_topo_config['dut_type'] = topo_definition['configuration_properties']['common']['dut_type']
             vm_topo_config['dut_asn'] = dut_asn
 
+        if hwsku == 'Cisco-8111-O64':
+            if 't1' in topo_name:
+                vm_topo_config['dut_type'] = "BackEndLeafRouter"
```

Comment:
what if we don't set this field to BackEnd[Leaf/tor]Router, any impact?

Reply:
**otherwise have to to hardcode hwsku to minigraph_png.j2 file** 

https://github.com/sonic-net/sonic-mgmt/blob/a42cf1d2ae00148035bb4b954c6871fd41d65000/ansible/templates/minigraph_png.j2#L106C57-L106C57

to make sure build correct device type for Cisco-8111-O64

<Device i:type="BackEndLeafRouter">
or 
<Device i:type="BackEndToRRouter">